src/components/WidgetVideo/WidgetVideo.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function WidgetVideoPresentation({ widgetData }) {
+  const { caption, video } = widgetData.settings
+
+  const getVideoElement = () =>
+    video ? (
+      <video
+        className="hidden md:flex absolute z-10 w-auto h-auto min-w-full min-h-full max-w-none"
+        autoPlay
+        muted
+        loop
+        src={video}
+      />
+    ) : null
+
+  const getCaptionElement = () =>
+    caption ? <div className="flex flex-wrap">{caption}</div> : null
+
+  return (
+    <section className="w-full" data-testid="WidgetVideoPresentation">
+      <div className="rounded overflow-hidden flex flex-col mx-auto max-w-2xl">
+        <div className="flex flex-col mx-auto overflow-hidden px-6 md:px-0 pt-6">
+          {getVideoElement()}
+          {getCaptionElement()}
```

Comment:
For the next PR you might want to do these inline without extracting them into functions:
`{caption && (<div className="flex flex-wrap">{caption}</div>)}`